osbuild/host.py

```diff
@@ -349,14 +349,20 @@ def call(self, method: str, args: Optional[Any] = None) -> Any:
         return ret
 
     def call_with_fds(self, method: str,
-                      args: Optional[Any] = None,
```

Comment:
Just for completeness: I most often did not do `List[T]` because it is deprecated with Python 3.9 but we cannot use the new syntax since we need to support Python 3.6 for RHEL 8 and 7.

Reply:
Yeah, I was not suggesting to change it, just explaining why I hesitated.